src/tests/factories/result-error-record-factory.ts

```diff
@@ -0,0 +1,24 @@
+import { ResultErrorRecord } from "../..//view-models/result-error-record";

```

Comment:
This path contains a double slash - looks like it's not failing anything, but it doesn't seem right. Could you quick update this, @mrjones2014?

Reply:
oops 😬 I'll fix it